gluoncv/utils/compress_json.py

```diff
@@ -0,0 +1,55 @@
+# pylint: disable=line-too-long
+"""Encode/Decode helper function for compressed quantized models"""
+import zlib
+import base64
+
+_compressed_int8_json = {
```

Comment:
I feel we should have the compressed string in another file or other files under `model_zoo`. Utils are utils, and different from specific model definitions. 

@zhreshold what do you suggest?

Reply:
agree, `model_zoo` is a better place for compressed strings

Reply:
@wuxun-zhang  Let's put the strings under `gluoncv/model_zoo/quantized/`. In the next PR we can replace all the current `.json` files with compressed strings.

Reply:
Fix.